fab/fabfile.py

```diff
@@ -638,18 +645,95 @@ def _deploy_without_asking():
         # hard update of manifest.json since we're about to force restart
         # all services
         _execute_with_timing(update_manifest)
```

Comment:
Yeah, the reason it was called twice was due to the irregularity of the workers stopping and swapping over. I think with this new deploy process, once should be ok